uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/feature/SpringMeteringFilterIT.java

```diff
@@ -46,6 +47,7 @@
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(classes = DefaultIntegrationTestConfig.class)
+@IfProfileValue(name = "spring.profiles.active", value = "nurego")
```

Comment:
Requirement is to execute metering test if active profiles **contains** `nurego`.  For example, if `spring.profiles.active` is set to multiple values as before "default,nurego".  Will using `@IfProfileValue` satisfy that requirement? Please confirm

`spring.profiles.active=deafult, nurego` - Test should execute
`spring.profiles.active=nurego` - Test should execute
`spring.profiles.active=deafult` - Test should NOT execute
`spring.profiles.active=deafult, some-other-profile` - Test should NOT execute